androidApp/src/main/java/com/example/tasky/android/agenda/components/DetailsAttendeeSectionAttendeeList.kt

```diff
@@ -0,0 +1,122 @@
+import com.example.tasky.model.agenda.Attendee
+import kotlinx.collections.immutable.ImmutableList
+import kotlinx.collections.immutable.toImmutableList
+
+@Composable
+fun DetailsAttendeeSectionAttendeeList(
+    @StringRes titleId: Int,
+    attendees: ImmutableList<Attendee>,
+    onDeleteIconClick: (String) -> Unit,
+    isEdit: Boolean,
+    creatorId: String,
+    modifier: Modifier = Modifier,
+) {
+    Column(modifier = modifier) {
+        Text(stringResource(titleId), style = typography.bodyMedium, lineHeight = 15.sp, color = DarkGray)
+        Spacer(modifier = Modifier.height(16.dp))
+        Column(verticalArrangement = Arrangement.spacedBy(4.dp)) {
+            attendees.map {
+                Row(
+                    modifier =
+                        Modifier
+                            .height(46.dp)
+                            .background(Light2, shape = RoundedCornerShape(10.dp))
+                            .padding(8.dp),
+                    verticalAlignment = Alignment.CenterVertically,
+                ) {
+                    Box(
+                        modifier =
+                            Modifier
+                                .size(32.dp)
+                                .background(Gray, CircleShape),
+                    ) {
+                        Text(
+                            getDisplayName(it.fullName),
+                            style = typography.bodyMedium,
+                            fontSize = 13.sp,
+                            lineHeight = 15.6.sp,
+                            color = Color.White,
+                            modifier =
+                                Modifier.align(
+                                    Alignment.Center,
+                                ),
+                        )
+                    }
+                    Spacer(modifier = Modifier.width(12.dp))
+                    Text(it.fullName, style = typography.labelLarge, lineHeight = 12.sp, color = DarkGray, modifier = Modifier.weight(1f))
+
+                    if (it.userId == creatorId) {
+                        Text(
+                            stringResource(R.string.creator),
+                            style = typography.labelLarge,
+                            lineHeight = 12.sp,
+                            color = LightBlue,
+                        )
+                    } else if (isEdit) {
+                        IconButton(onClick = { onDeleteIconClick(it.userId) }) {
+                            Icon(Icons.Outlined.Delete, contentDescription = null, tint = DarkGray)
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+private fun getDisplayName(fullName: String): String {
+    val splitName = fullName.trim().split(" ")
+    return when (splitName.size) {
+        0 -> ""
+        1 -> splitName[0].take(2)
+        else -> splitName[0][0].toString() + splitName.last()[0]
+    }
+}
```

Comment:
Hmm, this looks like business logic, wouldn’t you agree?
Placing this logic in your domain will allow you to easily test it with a light-weight unit test. Whereas now, you’d have to construct a UI test to validate this logic (and they can be clunky and are much slower than unit tests).